source/app/pages/index.tsx

```diff
@@ -26,7 +26,7 @@ class IndexPageBase extends React.Component<Props, IState> {
     constructor(props: Props) {
         super(props);
 
-        const cloudParam = (props.match.params as any).cloud || null;
+        const cloudParam = (props.match.params as any).cloud || "main";
```

Comment:
`"main"` should be an `export const` inside `shared/types.ts` since it's used here and `utilities.ts`